### PR TITLE
Feature/651 show notification usage

### PR DESCRIPTION
**Pull request by jozefbakus**
Implements: #651

### CODE REVIEW

File: src/test/scala/za/co/absa/hyperdrive/trigger/persistance/NotificationRuleRepositoryPostgresTest.scala [hunk 1]
```diff
@@ -162,6 +162,67 @@ class NotificationRuleRepositoryPostgresTest extends FlatSpec with Matchers with
...
+    val nr11 = nr.copy(workflowPrefix = Some("wor"), project = Some("PROJECT1"), id = 11)
+
+    await(db.run(workflowTable.forceInsertAll(Seq(w1, w2, w3, w4))))
+    await(db.run(notificationRuleTable.forceInsertAll(Seq(nr1, nr2, nr3, nr4, nr5, nr6, nr7, nr8, nr9, nr10, nr11))))
+
+    //Match on project
+    val result1 = await(notificationRuleRepository.getMatchingWorkflows(nr2.id))
+    result1 should contain theSameElementsAs Seq(w3)
+    //Match on project only, ignoring case
+    val result2 = await(notificationRuleRepository.getMatchingWorkflows(nr1.id))
```
Comment (kevinwallimann) [inline src/test/scala/za/co/absa/hyperdrive/trigger/persistance/NotificationRuleRepositoryPostgresTest.scala]: The order here is a little counter-intuitive. I would have expected `result1` for `nr1` and `result2` for `nr2`. Did you do that on purpose?

File: ui/src/app/components/admin/notification-rules/notification-rules-home/notification-rules-home.component.html [hunk 2]
```diff
@@ -102,6 +102,19 @@
     </clr-dg-cell>
   </clr-dg-row>
 
+  <ng-template clrIfDetail let-notificationRuleUsage (clrIfDetailChange)="onNotificationRuleUsageOpenClose($event)">
```
Comment (kevinwallimann) [inline ui/src/app/components/admin/notification-rules/notification-rules-home/notification-rules-home.component.html:105]: Is it somehow possible to show the first two columns when the detail view is expanded? Would make most sense if project and workflow prefix is shown, not only the project

Comment (jozefbakus) [reply ui/src/app/components/admin/notification-rules/notification-rules-home/notification-rules-home.component.html:105]: I am not sure but if you take a look couple of lines bellow, you can see that I do show it:
```
<clr-dg-detail-header>
        <strong>Project: </strong>{{notificationRuleUsage.project}}<br/>
        <strong>Workflow Prefix: </strong>{{notificationRuleUsage.workflowPrefix}}<br/>
        <strong>Matching workflows: </strong>
</clr-dg-detail-header>
```

File: src/test/scala/za/co/absa/hyperdrive/trigger/persistance/NotificationRuleRepositoryPostgresTest.scala (hunk 1 above)
Comment (jozefbakus) [reply src/test/scala/za/co/absa/hyperdrive/trigger/persistance/NotificationRuleRepositoryPostgresTest.scala]: Fixed

File: ui/src/app/components/admin/notification-rules/notification-rules-home/notification-rules-home.component.html (hunk 2 above)
Comment (kevinwallimann) [reply ui/src/app/components/admin/notification-rules/notification-rules-home/notification-rules-home.component.html:105]: Ok, I meant in the grid on the left side. But anyway, if it's not possible, so be it.

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**